### DISCUSSION TITLE
How to make Amaze default app

### DISCUSSION THREAD

**Discussion by qp5** [TeamAmaze/AmazeFileManager#3860] [Q&A]
Thankyou for Amaze file manager.
Anyone know how to make Amaze default file manager?

Option doesn't appear within: Settings / Apps / Default apps

**Comment by VishnuSanal** [comment]
Hi, thanks for reaching out to us. Did you mean that you want to make Amaze rhe default file manager? You can do that from settings -> apps. You may want to clear the default one, if one's already set.

**Reply by qp5** [reply]
Hi,

myAndroid pixel 7, uses 'Androids file manager' as default, 
i can disable it, though option to set 'Amaze file manager' as default doesn't exist?

**Settings: Apps**
 -  shows recent list of Apps, and a link to Default Apps,
 -  from Recent Apps list, Amaze File Manager appears, though no option to make Default app appears anywhere?
Eg
here is also how my screen looks: 
https://www.androidauthority.com/change-default-apps-android-2-633572/
 - File manager option doesn't appear!

 Thankyou

**Reply by VishalNehra** [reply]
Can you share screenshot of your default apps screen or a screen record of the issue you're talking about, and we'll take a look :)

**Reply by qp5** [reply]
hi, yes, 
Settings look same as hyperlink:
Not seeing option to set a file mgr.

![-Android-13-3-scaled](https://github.com/TeamAmaze/AmazeFileManager/assets/56560115/344c7b43-d454-401e-b717-b773e2b48669)

**Reply by VishnuSanal** [reply]
Unfortunately, we can't change the default file manager on newer Android versions it seems. :(
